judgments/models.py

```diff
@@ -119,3 +119,9 @@ class Meta:
         namespaces = {"search": "http://marklogic.com/appservices/search"}
 
     transform_to_html = xmlmodels.XsltField(join(dirname(__file__), "search_match.xsl"))
+
+
+class CourtDates(models.Model):
```

Comment:
not a concern for now - but do we envisage adding other court info to the db?

if so, we may consider generalising this class to something like `Court`.

Reply:
Aha, so, i started a spike for moving all the metadata into the DB and it complicated things considerably, so decided to keep this PR more specific. Agree that it's probably a good medium-term aim though!